R/derive_vars_crit.R

```diff
@@ -2,145 +2,12 @@
-#'   "999999-000001", 10, "IMMUNOLOGY", "J0033VN", 2, 4,
-#'   "999999-000001", 10, "IMMUNOLOGY", "I0019NT", 3, 6,
-#'   "999999-000001", 10, "IMMUNOLOGY", "M0019LN", 4, 4,
-#'   "999999-000001", 10, "IMMUNOLOGY", "R0003MA", 3, 6,
-#'   "999999-000001", 30, "IMMUNOLOGY", "J0033VN", 60, 4,
-#'   "999999-000001", 30, "IMMUNOLOGY", "I0019NT", 567, 6,
-#'   "999999-000001", 30, "IMMUNOLOGY", "M0019LN", 659, 4,
-#'   "999999-000001", 30, "IMMUNOLOGY", "R0003MA", 250, 6,
-#'   "999999-000002", 10, "IMMUNOLOGY", "J0033VN", 2, 4,
-#'   "999999-000002", 10, "IMMUNOLOGY", "I0019NT", 7, 6,
-#'   "999999-000002", 10, "IMMUNOLOGY", "M0019LN", 5, 4,
-#'   "999999-000002", 10, "IMMUNOLOGY", "R0003MA", 3, 6,
-#'   "999999-000002", 30, "IMMUNOLOGY", "J0033VN", 55, 4,
-#'   "999999-000002", 30, "IMMUNOLOGY", "I0019NT", 89, 6,
-#'   "999999-000002", 30, "IMMUNOLOGY", "M0019LN", 990, 4,
-#'   "999999-000002", 30, "IMMUNOLOGY", "R0003MA", 340, 6,
-#'   "999999-000003", 10, "IMMUNOLOGY", "J0033VN", 3, 4,
-#'   "999999-000003", 10, "IMMUNOLOGY", "I0019NT", 6, 6,
-#'   "999999-000003", 10, "IMMUNOLOGY", "M0019LN", 2, 4,
-#'   "999999-000003", 10, "IMMUNOLOGY", "R0003MA", 2, 6,
-#'   "999999-000003", 30, "IMMUNOLOGY", "J0033VN", 45, 4,
-#'   "999999-000003", 30, "IMMUNOLOGY", "I0019NT", 381, 6,
-#'   "999999-000003", 30, "IMMUNOLOGY", "M0019LN", 542, 4,
-#'   "999999-000003", 30, "IMMUNOLOGY", "R0003MA", NA, 6
-#' )
-#'
-#'
-#' derive_vars_crit(
-#'   dataset = input,
-#'   prefix = "CRIT1",
-#'   crit_label = "Titer >= ISLLOQ",
-#'   condition = !is.na(AVAL) & !is.na(ISLLOQ),
-#'   criterion = AVAL >= ISLLOQ
-#' )
-#'
-derive_vars_crit <- function(dataset, prefix, crit_label, condition, criterion) {
-  condition <- assert_filter_cond(enquo(condition))
-  criterion <- assert_filter_cond(enquo(criterion))
-
-  var_char <- paste0(prefix, "FL")
-  var_num <- paste0(prefix, "FN")
-
-  if (grepl("CRIT", prefix)) {
-    data <- dataset %>%
-      mutate(
-        `:=`(
-          !!var_char,
-          case_when(
-            !(!!condition) ~ NA_character_,
-            !!criterion & !!condition ~ "Y",
-            !(!!criterion) & !!condition ~ "N"
-          )
-        ),
-        `:=`(
-          !!var_num,
-          case_when(
-            !(!!condition) ~ NA_real_,
-            !!criterion & !!condition ~ 1,
-            !(!!criterion) & !!condition ~ 0
-          )
-        ),
-        `:=`(
-          !!prefix,
-          case_when(
-            !(!!condition) ~ NA_character_,
-            !!criterion & !!condition ~ crit_label,
-            !(!!criterion) & !!condition ~ crit_label
-          )
-        )
-      )
-  } else {
-    data <- dataset %>%
-      mutate(
-        `:=`(
-          !!var_char,
-          case_when(
-            !(!!condition) ~ NA_character_,
-            !!criterion & !!condition ~ "Y",
-            !(!!criterion) & !!condition ~ "N"
-          )
-        )
-      )
-  }
-
-  return(data)
-}
+#' @keywords deprecated
+#' .
```

Comment:
Hi @federicobaratin Thanks for making the changes. Instead of completely removing the function please adhere to the deprecation strategy for admiral packages as mentioned here https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation

Reply:
Hi Sukalpo, I have updated the function description.
I followed the strategy, by updating relative fields (function description, family and keywords) and removed examples, as suggested. Thanks!

Reply:
Hi @ahasoplakus, I have corrected most of the errors and now both ADIS template and vignette are fine. Same for the admiralvaccine function that need to be deprecated.
I think one link need to be updated in ADFACE. If you provide me the correct one, I can update it for you.
The other issue is that ADIS template does not recognize the new admiral function.
Please, let me know :)